src/controllers/organizations.js

```diff
@@ -0,0 +1,176 @@
+const { default: mongoose } = require('mongoose');
+const { Organization, User } = require('../models/user');
+const Fund = require('../models/fund');
+
+async function getOneOrganization(req) {
+    const { id: orgaId } = req.params;
+    const requiredUserFields = [
+        'id',
+        'firstName',
+        'lastName',
+        'email',
+        'profileImage',
+    ];
+    const requiredFundFields = [
```

Comment:
I think this variable and also the event one not really important, since you already setup the schema to have time stamp and virtule fields, so populating using just  'createdFunds' fields it's enough and i tried that with my get user/orga routes.

Reply:
i mean since you really selecting all fund and event fields